### PR TITLE
Updates to Ansible Bootstrap Role

### PR DESCRIPTION
**Pull request by claudiol**
* Added Ansible role info to README.md
* Fixed variable in defaults for Ansible Role
* Updated tasks names in verify-seed.yaml

### CODE REVIEW

File: ansible/roles/bootstrap-industrial-edge/README.md [hunk 1]
```diff
@@ -1,31 +1,73 @@
...
 
-Role Variables
---------------
+| Tool | Description | Download link |
+| ----------- | ----------- | ----------- |
+| kubernetes.core | The collection includes a variety of Ansible content to help automate the management of applications in Kubernetes and OpenShift clusters, as well as the provisioning and maintenance of clusters themselves | **ansible-galaxy collection install kubernetes.core** |
+| Kubernetes Python Cli | The kubernetes.core collection requires the Kubernetes Python client to interact with Kubernetes' APIs. | **pip3 install kubernetes** |
+| Python 3 | Python2 is deprecated from 1st January 2020. Please switch to Python3. | RHEL: <br> **yum -y install rh-python36** |
+| Vault CLI | The Vault CLI is a single static binary. It is a thin wrapper around the HTTP API. Every CLI command maps directly to the HTTP API internally. | RHEL: <br> **sudo yum-config-manager --add-repo https://rpm.releases.hashicorp.com/RHEL/hashicorp.repo**<br> **sudo yum -y install vault** <br><br> Fedora: <br> **sudo dnf config-manager --add-repo https://rpm.releases.hashicorp.com/fedora/hashicorp.repo** <br> **sudo dnf -y install vault** |
```
Comment (mbaldessari) [inline ansible/roles/bootstrap-industrial-edge/README.md:18]: I told the same to William on slack, but I believe this is not correct. We invoke the vault command from inside the vault container, so there is no need for the vault CLI on our laptops.

File: ansible/roles/bootstrap-industrial-edge/README.md [hunk 2]
```diff
@@ -1,31 +1,73 @@
...
+| component: | Name of the component to deploy |datacenter |                                                       
+| secret_name: | Name of the ArgoCD secret in OpenShift | "argocd-env" |                                                     
+| values_secret: | Location of the values-secret.yaml file | "{{ lookup('env', 'HOME') }}/values-secret.yaml" |                                                                    | values_global: | Location of the values-global.yaml file | "{{ pattern_repo_dir }}/values-secret.yaml" |             
+| kubeconfig: | Environment variable for KUBECONFIG | "{{ lookup('env', 'KUBECONFIG') }}"|
+| vault_init_file: | Init Vault file which will contain Vault tokens etc | "{{ pattern_repo_dir }}/common/pattern-vault.init"|                      
+| vault_ns: | Namespace for Vault | "vault"|                                                        
+| vault_pod: | Name of the initial Vault pod | "vault-0"|                                                       
+| vault_path: | Path to the Vault secrets for the Hub | "secret/hub"|                                                           
+| debug: | Whether or not to display debug info | False |
```
Comment (mbaldessari) [inline ansible/roles/bootstrap-industrial-edge/README.md:39]: If you do another pass at this maybe remove the whitespaces at the end

**Review by mbaldessari — COMMENTED** [review]
I suggested a couple of tweaks, we can add them on top later or change this PR, whichever you prefer ;)

File: ansible/roles/bootstrap-industrial-edge/README.md (hunk 1 above)
Comment (claudiol) [reply ansible/roles/bootstrap-industrial-edge/README.md:18]: Indeed @mbaldessari. Thanks for clarifying and reviewing this.  I will exclude it from the README file.

File: ansible/roles/bootstrap-industrial-edge/README.md [hunk 3]
```diff
@@ -1,31 +1,73 @@
...
+  hosts: localhost
+  connection: local
+  roles:
+    - bootstrap-industrial-edge
+```
+
+To start the execution of the role execute the following:
+
+```sh
+claudiol@fedora:industrial-edge (claudiol-ansible-updates *)
```
[the remark this answers is not in the document]
Comment (claudiol) [reply ansible/roles/bootstrap-industrial-edge/README.md:65]: Got it.  Thanks.

File: ansible/roles/bootstrap-industrial-edge/README.md (hunk 2 above)
Comment (claudiol) [reply ansible/roles/bootstrap-industrial-edge/README.md]: Completed.

File: ansible/roles/bootstrap-industrial-edge/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (claudiol) [reply ansible/roles/bootstrap-industrial-edge/README.md]: Completed

**Review by mbaldessari — APPROVED** [review]

**Pull request merged**